package.json

```diff
@@ -3,8 +3,11 @@
   "version": "1.2.0",
   "description": "Common code (API client, Redux stores, logic, utility functions) for building a Mattermost client",
   "homepage": "https://github.com/mattermost/mattermost-redux",
```

Comment:
With the correct repo link this could now be used to point to the main MM website or some developer docs related to mm-redux